src/App.js

```diff
@@ -1,5 +1,78 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+function printResult(sum){
+  return MissionUtils.Console.print(`결과 : ${sum}`);
+}
+
+function sumNumber(numberArr){
+  let sum = 0;
+  for(var number of numberArr) sum += number;
+  return sum;
+}
+
+function extractNumber(input, splitunit){
+  if(input.indexOf("\\n") != -1)
```

Comment:
```suggestion
  if(input.indexOf("\\n") !== -1)
```
별 다른 이유가 없다면 일치 비교를 사용해서 엄격하게 비교하는게 좋을 것 같아요!